askama_parser/src/lib.rs

```diff
@@ -476,7 +476,7 @@ fn num_lit<'a>(i: &mut &'a str) -> ParseResult<'a, Num<'a>> {
         }
     };
 
-    let num = if let Ok(Some(num)) = opt(int_with_base.take()).parse_next(i) {
+    let num = if let Some(num) = opt(int_with_base.take()).parse_next(i)? {
```

Comment:
It took me like 2 hours to delete 4 characters and to insert 1 character. -_-

Reply:
2 hours well spent then. :)